### PR TITLE
Add auto_dj project

### PR DESCRIPTION
**Pull request by CrazyByDefault**

### CODE REVIEW

File: data/projects/auto_dj.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+This is the feature list -
+
+-  Music classification based on mood,
+-  Tempo/signature recognition,
+-  User preference,
+-  Music prediction based on the above.
+-  Adding additional beats/drops to a single track based on it's features
+-  Mixing the next song with the first, _"like a DJ would"._
+
+
```
Comment (mishra-sid) [inline data/projects/auto_dj.md]: Add a section about what the student has to propose.

Comment (mishra-sid) [reply data/projects/auto_dj.md]: Also add atleast a few Technical details on which the student should have an insight on.

**Review by mishra-sid — COMMENTED** [review]
Added few points

File: data/projects/auto_dj.md (hunk 1 above)
Comment (CrazyByDefault) [reply data/projects/auto_dj.md]: Those are there in requirements I guess

**Pull request merged**